webapp/utils/watch/evmDeposits.ts

```diff
@@ -13,10 +40,22 @@ export const watchEvmDeposit = async function (deposit: EvmDepositOperation) {
   if (!receipt) {
     return updates
   }
-  const newStatus =
-    receipt.status === 'success'
-      ? EvmDepositStatus.DEPOSIT_TX_CONFIRMED
-      : EvmDepositStatus.DEPOSIT_TX_FAILED
+
+  const status = await crossChainMessenger.getMessageStatus(
+    deposit.transactionHash,
+    0,
+    deposit.direction,
+  )
+
+  let newStatus: EvmDepositStatus
+  if (receipt.status === 'success' && status === MessageStatus.RELAYED) {
```

Comment:
strictly speaking, I think we can remove this check

```suggestion
  if (status === MessageStatus.RELAYED) {
```

If `status === MessageStatus.RELAYED`, then `receipt.status === 'success'` should always be true.

Reply:
Indeed. Simplified. Thanks!